.changeset/calm-peas-work.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": minor
+---
+
+feat: add line charts
```

Comment:
let's add bit more detailed changelogs going forward, this will be crucial in future when we want to do auto migrations. 

feat(blade): added Chart subcomponents & Line Chart component

[add the link to storybook]